citation.cff

```diff
@@ -3,12 +3,8 @@ message: "Please cite the following reference if you use the code for your publi
 authors:
 - family-names: "Jiang"
   given-names: "Chengxin"
-  orcid: "https://orcid.org/0000-0000-0000-0000?"
 - family-names: "Denolle "
   given-names: "Marine"
-  orcid: "https://orcid.org/0000-0000-0000-0000?"
 title: "NoisePy: a new high-performance python tool for seismic ambient noise seismology."
-version: 2.0.4?
-doi: 10.5281/zenodo.1234?
```

Comment:
@nrich20 what does this field do?

Reply:
@carlosgjs 
The version line specifies the version number of the software or research artifact being described. 
The doi line specifies a Digital Object Identifier (DOI) for the software or research artifact being described in the CFF file.